src/activities/drawletters/drawletters.qrc~

```diff
@@ -0,0 +1,48 @@
+<RCC>
```

Comment:
Hi, thanks for your suggestions . I want to ask one thing , I do not have to commit which file 1.) drawletters.qrc  or README or both ??